etl_server/pipelines/controllers.py

```diff
@@ -141,7 +141,8 @@ def query_pipelines(self, user=None, public=None):
             status = statuses.get(res['key'], status)
             res['value']['status'] = status
         query_results['result'] = list(map(lambda x: dict(x.get('value'), owner=x.get('owner'), private=x.get('private'), 
-                                    created_at=(x['created_at']).isoformat(), updated_at=(x.get('updated_at')).isoformat()), results))
+                                    created_at=((x.get('created_at')).isoformat() if x.get('created_at') != None else x.get('created_at')), 
```

Comment:
1. In Python it's more correct to use `is not None` instead of `!= None`
2. If it is `None` you can use `None` in the else clause instead of `x.get(...)`